django/contrib/postgres/fields/array.py

```diff
@@ -179,6 +181,19 @@ def as_sql(self, qn, connection):
 
 
 @ArrayField.register_lookup
+class ArrayIn(In):
```

Comment:
Is this new functionality or just something needed due to the new API? I assume the latter since I don't see any new tests, but just wanted to double check.